paperqa/docs.py

```diff
@@ -448,3 +448,18 @@ async def aquery(
         answer.references = bib_str
         answer.passages = passages
         return answer
+    
+    def chat(self):

```

Comment:
Let's move this to CLI, and not call it `chat`, since there is no memory of the conversation